lib/iron_trail/query_transformer.rb

```diff
@@ -22,7 +24,8 @@ def create_query_transformer_proc
         metadata = JSON.dump(current_metadata)
 
         if metadata.length > METADATA_MAX_LENGTH
-          # TODO: how to properly issue a warning here?
+          Rails.logger.warn("IronTrail metadata is longer than maximum length! #{metadata.length} > #{METADATA_MAX_LENGTH}")
```

Comment:
can we add an alert on mezmo to catch these?

Reply:
https://trusted-health.atlassian.net/browse/FND-3419